Directory.Build.props

```diff
@@ -207,20 +207,13 @@
   <!-- Import packaging props -->
   <Import Project="$(RepositoryEngineeringDir)Packaging.props" />
 
-  <!-- Set the kind of PDB to Portable and turn on SourceLink (fetching source from GitHub) -->
+  <!-- Set the kind of PDB to Portable -->
   <PropertyGroup>
     <!-- Always pass portable to override arcade sdk which uses embedded for local builds -->
     <DebugType>portable</DebugType>
 
     <!-- Empty DebugType when building for netfx and in windows so that it is set to full or pdbonly later -->
     <DebugType Condition="'$(TargetsNetFx)' == 'true' AND '$(OS)' == 'Windows_NT'" />
-
-    <!-- Rhel 6 and FreeBSD doesn't support the source control git package so disable SourceLink -->
-    <EnableSourceLink Condition="$(RuntimeOS.StartsWith('rhel.6')) OR '$(_runtimeOSFamily)' == 'FreeBSD'">false</EnableSourceLink>
-    <!-- is not supported on ARM platforms either. -->
-    <EnableSourceLink Condition="'$(HostArch)' == 'Arm' OR '$(HostArch)' == 'Arm64'">false</EnableSourceLink>
-    <!-- Disable source link on local builds. -->
-    <EnableSourceLink Condition="'$(ContinuousIntegrationBuild)' != 'true' and '$(OfficialBuildId)' == ''">false</EnableSourceLink>
```

Comment:
We don't want to enable source link locally as that doesn't play well with code coverage. Currently coverlet uses the source link information if it's present and there's no switch to opt-out.

Reply:
Is there an issue tracking this? Why isn't the reason stated in the comments?

Reply:
I started 2 project s,,,not knowing the effects it would have,,,I just want  me, my wife and kids back as a family,,,,For me to protect and provide,,,that's all I'm asking

Reply:
You are right, this deserves a comment. I had a tracking issue in coverlet to make sourcelink more configurable: https://github.com/tonerdo/coverlet/issues/312. The root problem here is that we want source link url paths for S.P.CoreLib but not for others which we live build.

Reply:
Thanks for the link. I'm not sure I understand though. What is coverlet using Source Link for? Wouldn't you want in general to link all source paths to their original URLs? Why would you want to link some and not others?

Reply:
Generating HTML reports (ie with ReportGenerator) based on the coverlet report with source link paths takes A VERY LONG time. We only want to resolve sources for System.Private.CoreLib (and fetch them) but not for the local ones in corefx.

Reply:
Interesting. I don't see why would application of Source Link mapping significantly increase the time. It's pretty much just a dictionary lookup.

Reply:
Viktor let me understand well the problem, you want filter out all local asm and use source link only for SPC because on corefx repo there aren't sources?And use source link for locals is slow due to download of sources?

Reply:
Exactly. Tools like ReportGenerator fetch all the remote sources (with a cache). Also a commit isn't available for source link until it's pushed.

Reply:
Can you share a link to the generated report? It's not clear to me why would it need to fetch all the sources eagerly. I'd expect it to display links that the user can dig into if they want to see the coverage for a specific file.

Reply:
Run dotnet msbuild /t:RebuildAndTest /p:Coverage on a test project, ie Regex.

Reply:
@ViktorHofer 
I have added the line back for now.

BTW, another question - why change `DebugType` from the Arcade defaults? Is it also for codecov?

Reply:
Thanks.

> BTW, another question - why change DebugType from the Arcade defaults? Is it also for codecov?

@ericstj?

Reply:
@weshaggard did this.  I suspect it was due to rewriting tools that required an actual PDB like GenFacades and ILLink.

Reply:
I don't recall exactly but I do kind of remember there were some issues with something requiring a pdb and not supporting embedded pdbs.

Reply:
Ok. Might be worth trying to remove this again, see what fails and add a comment/issue.